lib/transforms/caffeextension_legalizer.cc

```diff
@@ -386,19 +386,27 @@ static std::vector<Def> ConvertPriorBox(const CAFFEExtensionInst* ext,
           float ar_s = sqrtf(ar);
           float box_width = min_size * ar_s;
           float box_height = min_size / ar_s;
-          data.push_back(((center_x - box_width) / 2) / image_w);  // xmin.
-          data.push_back(((center_y - box_height) / 2) / image_h); // ymin.
-          data.push_back(((center_x + box_width) / 2) / image_w);  // xmax
-          data.push_back(((center_y + box_height) / 2) / image_h); // ymax.
+          data.push_back((center_x - std::round(box_width / 2)) /
```

Comment:
caffe doesn't round it. https://github.com/intel/caffe/blob/master/src/caffe/layers/prior_box_layer.cpp